lib/simplecov/result_merger.rb

```diff
@@ -54,20 +54,24 @@ def results
         results
       end
 
+      # Merge two or more SimpleCov::Results into a new one with merged
+      # coverage data and the command_name for the result consisting of a join
+      # on all source result's names
+      def merge_results(*results)
```

Comment:
I think you're right that there could be some confusion. I'm a little wary of calling this `resultsets` because this is an array of `SimpleCov::Result`s, not an array of `.resultset.json`-type data. That actually seems like a problem with `RawCoverage.merge_resultsets` too. But I'm not super familiar with SimpleCov's terminology, so maybe `resultsets` is an OK name here?

Without any concern for backward compatibility, I'd be tempted to rename `ResultMerger.results` to `ResultMerger.stored_results`, and leave the argument here as `*results`.